snippets/compose-functions.md

```diff
@@ -0,0 +1,15 @@
+### Compose functions
+
+Use the `...rest` operator to gather all function arguments into an array. Return a function which takes
+a single argument and uses `Array.reduceRight()` to return the result of applying each function.
+
+```js
+const compose = (...fns) => n => fns.reduceRight((acc, fn) => fn(acc), n);
```

Comment:
This is unary. Composition function are expected to be variadic and then unary, however it looks like this one is purely unary

Reply:
https://github.com/Chalarangelo/30-seconds-of-code/pull/141/files should show an easy fix for this. It's a little hard to understand, but it works like a charm!